spec/grape_log_formatter/basic_spec.rb

```diff
@@ -6,18 +6,23 @@
 
   let(:result) { GrapeLogFormatter::Basic.new.call(severity, time, nil, data) }
   let(:pid) { $PROCESS_ID }
+  let(:uniq_id) { "a345dv" }
   let(:exception) do
     e = StandardError.new("Application failed")
     e.set_backtrace(%w(file_1 file_2))
     e
   end
 
+  before do
+    allow(SecureRandom).to receive(:hex) { uniq_id }
+  end
+
   context "data is a string" do
     let(:data) { "Warning message" }
 
     it "contain time, severity and data string" do
       expect(result).to eq <<-STR
-[2016-05-06 18:20:00 +0200] INFO -- Warning message
+[2016-05-06 18:20:00 +0200] <a345dv:##{pid}> INFO -- Warning message
```

Comment:
Very minor, but could be `#{uniq_id}` instead of the raw string - minor point, not really too bothered about it :).